contrib/local/setup_tgrade.sh

```diff
@@ -16,28 +18,30 @@ for x in validator systemadmin; do
     (
       echo "$PASSWORD"
       echo "$PASSWORD"
-    ) | tgrade keys add validator
+    ) | tgrade keys add "$x"
   fi
 done
 
 # hardcode the account for this instance
 echo "$PASSWORD" | tgrade add-genesis-account systemadmin "1000000000$STAKE"
-echo "$PASSWORD" | tgrade add-genesis-account validator "1001000000$STAKE" --vesting-amount="1000000000$STAKE" --vesting-end-time="$(date -v+10y +%s)"
+echo "$PASSWORD" | tgrade add-genesis-account validator "1001000000$STAKE" --vesting-amount="1000000000$STAKE" --vesting-end-time="$(date -d "+10 years" +%s)"
```

Comment:
`$(date -d "+10 years" +%s)` does not work on OSX

Reply:
How about adding a random 10 year future date here? It does not have to be dynamic to make it work